opscode-pushy-client.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = PushyClient::VERSION
 
-  gem.required_ruby_version = '>= 2.4'
+  gem.required_ruby_version = '>= 2.5'
 
   gem.add_dependency "chef", ">= 13.0", "< 16.0"
```

Comment:
if we require Ruby 2.5 we should also require Chef 14+

Reply:
done